flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/JarRunHandler.java

```diff
@@ -92,7 +93,20 @@ public CompletableFuture<JarRunResponseBody> handleRequest(
             @Nonnull final DispatcherGateway gateway)
             throws RestHandlerException {
 
-        final Configuration effectiveConfiguration = new Configuration(configuration);
+        Map<String, String> requestJobFlinkConfig =
+                HandlerRequestUtils.fromRequestBodyOrQueryParameter(
+                        (request.getRequestBody()).getFlinkConfiguration(),
+                        () -> null,
+                        null,
+                        this.log);
+        Configuration effectiveConfiguration;
+        if (requestJobFlinkConfig != null) {
+            effectiveConfiguration = new Configuration();
+            requestJobFlinkConfig.forEach(effectiveConfiguration::setString);
```

Comment:
In this case we completely ignore `this.configuration` which doesn't correct.

Reply:
I am now changing it to
`Configuration effectiveConfiguration = new Configuration(this.configuration);`